src/main/java/io/cdap/plugin/gcp/speech/SpeechToTextTransform.java

```diff
@@ -314,49 +325,78 @@ public String getLanguage() {
     }
 
     @Nullable
-    public Integer getSampleRate() {
+    public Integer getSampleRate() throws IllegalArgumentException {
```

Comment:
methods generally do not declare the runtime exceptions that they throw and instead just document it in the javadoc.